docs/architecture/modernize-desktop/whats-new-dotnet-core.md

```diff
@@ -0,0 +1,144 @@
+
+This framework gives applications many benefits in terms of efficiency and performance, simplifying the packaging and deployment in the different supported platforms.
+
+The benefits of .NET Core come from these three characteristics:
+
+- **Cross-platform:** It allows application execution on different platforms (Windows, macOS, and Linux).
+- **Open source:** .NET Core platform is open source and available through GitHub, fostering transparency and community contributions.
+- **Supported:** Microsoft officially supports .NET Core.
+
+In .NET Core 3.0, besides the existing support for web and cloud, there's also support for desktop, IoT, and AI domains. The goal for this framework is impressive: to target every type of .NET development present and future. Microsoft plans to complete this vision with .NET 5 at the end of 2020. The "Core" name was removed to reinforce its uniqueness in the .NET world.
+
+![All domains of .NET 5](./media/whats-new-dotnet-core-desktop/all-domains-of-dotnet5.png)
+
+## .NET Framework vs. .NET Core
+
+So now that you understand the relevance of .NET Core inside the Microsoft strategy for .NET, you might be wondering what happens with .NET Framework. You could be asking questions like: do you have to abandon it? Is it going to disappear? What are my choices to modernize the applications I have on .NET Framework?
+
+In 2019 the last version of the **.NET Framework - 4.8** was released. It included three major improvements for desktop applications:
+
+- **Modern browser and media controls**: Today, .NET desktop applications use Internet Explorer and Windows Media Player for showing HTML and playing media files. Since these legacy controls don't show the latest HTML or play the latest media files, new controls were added that take advantage of Microsoft Edge and newer media players that support the latest standards.
+- **Access to UWP controls**: UWP contains new controls that take advantage of the latest Windows features and touch displays. You don't have to rewrite your applications to use these new features and controls, so you can use these new features in your existing WPF or Windows Forms code.
+- **High-DPI improvements**: The resolution of displays is increasing to 4K and 8K resolutions. So, .NET Framework 4.8 adds new HDPI improvements to make sure your existing Windows Forms and WPF applications can look great on these new displays.
+
+Since .NET Framework is installed on millions of machines, Microsoft will continue to support it but won't add new features.
+
+.NET Core is the open-source, cross-platform, and fast-moving version of .NET. Because of its side-by-side nature, it can take changes without the fear of breaking any application. This means that .NET Core will get new APIs and language features over time that .NET Framework won't. Also, **.NET Core** already has features, that were impossible for .NET Framework, such as:
```

Comment:
removed extra comma

```suggestion
.NET Core is the open-source, cross-platform, and fast-moving version of .NET. Because of its side-by-side nature, it can take changes without the fear of breaking any application. This means that .NET Core will get new APIs and language features over time that .NET Framework won't. Also, **.NET Core** already has features that were impossible for .NET Framework, such as:
```